server/py_files/models/LSTM/LSTMClassifier.py

```diff
@@ -14,22 +14,24 @@ class LSTMClassifier(KerasSentenceClassifier):
     def __init__(self, name, feature_type):
         super().__init__(name, feature_type)
         self.path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'trained', name + '_' + feature_type)
-        # self.labels_pred = None
+        self.max_len = 100
 
     def train(self, samples, labels):
         features = self.choose_features(samples, True)
 
         model = Sequential()
         embedding_size = 100
         if self.feature_type == 'word-embeddings':
-            x_train = pad_sequences(features, maxlen=100, padding='post')
+            x_train = pad_sequences(features, maxlen=self.max_len, padding='post')
             pretrained_embeddings = Embeddings(self.name, embedding_size).vectors()
             vocab_size = pretrained_embeddings.shape[0]
-            model.add(Embedding(vocab_size,100,input_length = 100, trainable =True, weights = [pretrained_embeddings] ))
+            model.add(Embedding(input_dim=vocab_size, output_dim=embedding_size,
```

Comment:
@astrebeck to cross check (have made sure we are named variables as inputs to avoid and confusion and things are declared/defined once for consistency)